src/quo/components/wallet/approval_label/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo.components.wallet.approval-label.view
+  (:require [oops.core :refer [oget]]
```

Comment:
We have 15 usages of `oops.core` using `:refer`. But ~60 usages of the namespace using `:as` because our guideline recommends avoiding `:refer` https://github.com/status-im/status-mobile/blob/b358c06a3132692c705e12cfd417a764e869341d/doc/new-guidelines.md#requireimport. It's better if we can stay consistent.